webinar.html

```diff
@@ -0,0 +1,231 @@
+<li><strong>2019/09/05</strong>: <a href="https://www.youtube.com/playlist?list=PLe5rNUydzV9QHe8VDStpU0o8Yp63OecdW">Introduction on deep Bayesian methods</a></li>
+<li><strong>2019/08/01:</strong>&nbsp;<a href="http://A new perspective on adversarial-perturbations">A new perspective on adversarial-perturbations</a></li>
+<li><strong>2019/07/11-18:</strong>&nbsp;<a href="https://www.youtube.com/watch?v=KDRN-FyyqK0">ICML 2018: Tutorial Session: Toward the Theoretical Understanding of Deep Learning</a></li>
+<li><strong>2019/07/04</strong>: <a href="https://www.youtube.com/watch?v=z1xs9jdZnuY">Attention is all you need</a></li>
+<li><strong>2019/06/27:</strong>&nbsp;<a href="https://cbmm.mit.edu/video/principles-and-applications-relational-inductive-biases-deep-learning">Principles and applications of relational inductive biases in deep learning</a>&nbsp;by&nbsp;Kelsey Allen</li>
+<li><strong>2019/06/13:</strong> <a href="https://www.youtube.com/watch?v=CrkGMnh7KiA">Active learning</a></li>
+<li><strong>2019/05/23-30 â 06/06</strong>:&nbsp; <a href="https://www.youtube.com/watch?time_continue=2333&amp;v=fQyT5wkFxGI">Adversarial robustness : theory and practice</a>. (NeuRIPS tutorial 2018)</li>
+<li><strong>2019/05/16</strong>:&nbsp; <a href="https://www.youtube.com/watch?v=TB07_mUMt0U">Deep learning determinism</a>. by&nbsp;Duncan Riach</li>
+<li><strong>2019/05/09</strong>:<a href="https://www.youtube.com/watch?v=e5othcNmync">&nbsp;Guarding user Privacy with Federated Learning and Differential Privacy </a>by B. McMahan</li>
+<li><strong>2019/05/02:</strong>&nbsp;<a href="https://www.youtube.com/watch?v=0eBR8a4MQ30">Automatic Machine Learning</a> (NeurIPS 2018 Turorial) part2</li>
+<li><strong>2019/05/02:</strong><a href="https://www.youtube.com/watch?v=0eBR8a4MQ30">&nbsp;Automatic Machine Learning</a> (NeurIPS 2018 Tutorial) part 1</li>
+<li><strong>2019/04/04</strong>: Introduction to the new âBelugaâ GPU server</li>
+<li><strong>2019/03/28</strong>: <a href="https://livestream.com/newyorkacademyofsciences/ml2019-3/videos/188158627">Integrating Domain-Knowledge into deep learning</a>&nbsp;by Ross Salakoudenov</li>
+<li><strong>2019/03/21</strong>: Variational inference, lower bound and VAE</li>
+<li><strong>2019/03/14</strong>:&nbsp;<a href="http://videolectures.net/DLRLsummerschool2018_wilson_bayesian_neural_nets/">Bayesian NN from Andew Gordon Wilson</a>&nbsp; (part 3)</li>
+<li><strong>2019/02/28</strong>:&nbsp;<a href="http://videolectures.net/DLRLsummerschool2018_wilson_bayesian_neural_nets/">Bayesian NN from Andew Gordon Wilson</a>&nbsp; (part 2)</li>
+<li><strong>2019/02/21</strong>:&nbsp;<a href="http://videolectures.net/DLRLsummerschool2018_wilson_bayesian_neural_nets/">Bayesian NN from Andew Gordon Wilson</a>&nbsp; (part 1)</li>
+<li><strong>2019/02/07:</strong>&nbsp;<a href="https://www.facebook.com/nipsfoundation/videos/539369239863523/">Designing Computer Systems for Software 2.0</a>, by Kunle Olukotun, NeuRIPS 2018</li>
```

Comment:
For Facebook, the links can be embedded like follows:
```suggestion
<li><strong>2019/02/07:</strong>&nbsp; Designing Computer Systems for Software 2.0, by Kunle Olukotun, NeuRIPS 2018</li>
<iframe src="https://www.facebook.com/plugins/video.php?height=314&href=https%3A%2F%2Fwww.facebook.com%2Fnipsfoundation%2Fvideos%2F539369239863523%2F&show_text=false&width=560" width="560" height="314" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowfullscreen="true" allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share" allowFullScreen="true"></iframe>
```